gnd/src/main/java/com/google/android/gnd/rx/Loadable.java

```diff
@@ -98,6 +85,15 @@ public boolean isLoaded() {
         .startWith(Loadable.loading());
   }
 
+  public LoadState getState() {
+    return state;
+  }
+
+  public boolean isLoaded() {
+    return state == LoadState.LOADED;
+  }
```

Comment:
In the future, please reorder method in a separate PR if possible, since mixing actual changes with reorders makes it hard to tell what was changed vs moved.

Reply:
Sure